data/DSLAB/projects.yaml

```diff
@@ -189,9 +234,30 @@ projects:
         tags:
             - Cloud
             - Performance
+        information:
+          - type: Paper
+            title: "Parallel Symbolic Execution for Automated Real-World Software Testing"
+            url: http://dslab.epfl.ch/pubs/cloud9.pdf
+            notes:
+              - label: Published in
+                text: "EuroSys 2011"
+                url: http://eurosys2011.cs.uni-salzburg.at/pdf/eurosys2011-salomie.pdf
+          - type: Paper
+            title: "Automated Software Testing as a Service"
+            url: http://dslab.epfl.ch/pubs/cloud9.pdf
```

Comment:
I believe this is the wrong link (a copy/paste from the previous one — yes, it is wrong on the Cloud9 page as well).
The correct link is: https://dslab.epfl.ch/pubs/taas.pdf